stable/k8s-secret-updater/README.md

```diff
@@ -0,0 +1,51 @@
+# Kubernetes Secret Updater
+
+This is used in conjunction with Confidant, to update the secrets in k8s namespaces.
```

Comment:
Please add a link to https://github.com/fairfaxmedia/k8s-secret-updater

Reply:
... and https://hub.docker.com/r/fairfaxmedia/k8s-secret-updater - there's some good doco there :)

Reply:
Added!